### PR TITLE
Update README to mention Makefile variables

### PR DESCRIPTION
**Pull request by rhatdan**
Also mention bootc-image-builder target.

### CODE REVIEW

**Comment by rhatdan** [comment]
This will need to be updated for all recipe readmes.

File: recipes/natural_language_processing/chatbot/README.md [hunk 1]
```diff
@@ -143,3 +143,29 @@ image registry with a single `bootc` command. This works especially well for fle
...
+| `qcow2` **(default)** | [QEMU](https://www.qemu.org/)                                                         |
+| `vmdk`                | [VMDK](https://en.wikipedia.org/wiki/VMDK) usable in vSphere, among others            |
+| `anaconda-iso`        | An unattended Anaconda installer that installs to the first disk found.               |
+| `raw`                 | Unformatted [raw disk](https://en.wikipedia.org/wiki/Rawdisk).                        |
+
+The Makefile can be used to automatically generate a disk image from the bootc
+image. The following command will create an qcow2 image file from the default
+bootc image in the build subdirectory.
+
+`make bootc-image-builder DISK_TYPE=qcow2`
```
Comment (sallyom) [inline recipes/natural_language_processing/chatbot/README.md]: we added the target `bootc-image-builder`? 👀

Comment (sallyom) [reply recipes/natural_language_processing/chatbot/README.md]: I see we have 😃 🥇

**Comment by rhatdan** [comment]
@MichaelClifford @sallyom @lmilbaum @Gregory-Pereira PTAL

**Review by Gregory-Pereira — APPROVED** [review]
/lgtm, very nice touch with the `recipes/common/README.md` file

**Pull request merged**